### PR TITLE
Context aware

### PR DESCRIPTION
**Pull request by YZYLAB**
Necessary for Electron 14.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "zeromq",
-  "version": "5.2.8",
+  "version": "5.2.9",
```
Comment (aminya) [inline package.json:3]: Could you revert this?

**Review by aminya — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (YZYLAB) [reply package.json]: I think its fixed now

**Comment by nbabanov** [comment]
@JustinTulloss @lgeiger

**Comment by captainsafia** [comment]
Stepping into this with no context (pun intended).

> Necessary for Electron 14.0

Where is this documented?

Also, how do we validate this fix?

Not opposed to it on principle, but want to make sure I's are dotted and T's are crossed.

**Comment by YZYLAB** [comment]
https://github.com/electron/electron/issues/18397 

Here they explained what they pushed in Electron 14.0

**Comment by nbabanov** [comment]
@captainsafia Could you provide any info?

**Review by aminya — APPROVED** [review]

**Pull request merged**